ui/api/resources/user.js

```diff
@@ -1,3 +1,4 @@
+import regeneratorRuntime from "regenerator-runtime";
```

Comment:
I added this line when I couldn't get the users page to load due to a missing import when using the async calls. I can't reproduce the error I was getting now though so it might have just been an issue with my environment. I've removed the import.